Source/Diagnostics/FlushFormats/FlushFormatAscent.cpp

```diff
@@ -16,16 +16,25 @@ FlushFormatAscent::WriteToFile (
     amrex::Vector<amrex::Geometry>& geom,
     const amrex::Vector<int> iteration, const double time,
     const amrex::Vector<ParticleDiag>& particle_diags, int nlev,
-    const std::string prefix, int /*file_min_digits*/, bool plot_raw_fields,
+    const std::string prefix, int file_min_digits, bool plot_raw_fields,
     bool plot_raw_fields_guards,
     const bool /*use_pinned_pc*/,
-    bool /*isBTD*/, int /*snapshotID*/, const amrex::Geometry& /*full_BTD_snapshot*/,
+    bool isBTD, int snapshotID, const amrex::Geometry& /*full_BTD_snapshot*/,
     bool /*isLastBTDFlush*/, const amrex::Vector<int>& /* totalParticlesFlushedAlready*/) const
 {
 #ifdef AMREX_USE_ASCENT
     WARPX_PROFILE("FlushFormatAscent::WriteToFile()");
-
     auto & warpx = WarpX::GetInstance();
+    int file_iter;
+    if (!isBTD)
+    {
+      file_iter = iteration[0];
+    } else
+    {
+      file_iter = snapshotID;
+    }
```

Comment:
I think this can be coded in one line (same for similar code in the other files):
```suggestion
    const int file_iter = (isBTD) ? snapshotID : iteration[0];
```

Reply:
A good idea at the time... now the logic's more complicated